docs/apm/troubleshooting.asciidoc

```diff
@@ -15,6 +15,7 @@ don't forget to check our other troubleshooting guides or discussion forum:
 * {apm-server-ref}/troubleshooting.html[APM Server troubleshooting]
 * {apm-dotnet-ref}/troubleshooting.html[.NET agent troubleshooting]
 * {apm-go-ref}/troubleshooting.html[Go agent troubleshooting]
+* iOS agent troubleshooting coming soon
```

Comment:
Link to new iOS troubleshooting topic added in https://github.com/elastic/apm-agent-ios/pull/45.

```suggestion
* {apm-ios-ref}/troubleshooting.html[iOS agent troubleshooting]
```